src/node/lattice.rs

```diff
@@ -212,14 +214,16 @@ impl ExecutionLattice {
             let mut demoted_leaves: Vec<NodeIndex> = Vec::new();
             // These edges are redundant and must be removed.
             let mut redundant_edges: Vec<EdgeIndex> = Vec::new();
-            // Iterate through the forest with a DFS from each leaf to figure out where to add dependency edges.
+            // Iterate through the forest with a DFS from each leaf to figure out where to add 
```

Comment:
Are these changes made by rustfmt? If so, could you upgrade Rust to >= 1.52.0 nightly to make sure our formatting is consistent?

Reply:
I'm on 1.53.0. I changed these because the original did not fit within 100 columns. I'll RustFmt this again.